bin/MadGraph5_aMCatNLO/cards/production/13TeV/VV/WZTo2Q2L_4f_amcatnloFxFx/WZTo2Q2L_4f_amcatnloFxFx_madspin_card.dat

```diff
@@ -0,0 +1,16 @@
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+set Nevents_for_max_weight 250 # number of events for the estimate of the max. weight
```

Comment:
use "Nevents_for_max_weigth" for MG265, same for other cards

Reply:
Done